sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/cpd/internal/DefaultCpdTokens.java

```diff
@@ -87,7 +88,7 @@ public DefaultCpdTokens addToken(TextRange range, String image) {
     if (excluded) {
       return this;
     }
-    Preconditions.checkState(lastRange == null || lastRange.end().compareTo(range.start()) <= 0,
+    checkState(lastRange == null || lastRange.end().compareTo(range.start()) <= 0,
```

Comment:
If the goal is to remove Guava, you should replace checkState by a if(...) throw ISE